accounts/migrations/0034_auto_20170911_1832.py

```diff
@@ -14,6 +14,7 @@ class Migration(migrations.Migration):
     operations = [
         migrations.AlterModelOptions(
             name='user',
-            options={'ordering': ['username'], 'permissions': (('can_create_users', 'Can Create User'), ('can_view_users', 'Can View User'), ('can_edit_users', 'Can Edit User'), ('can_remove_users', 'Can Remove User'), ('can_view_user_permissions', 'Can View User Permissions'), ('can_edit_user_permissions', 'Can Edit User Permissions'), ('can_read_all_user_data', 'Can Read All User Data'))},
+            options={'ordering': ['username'], 'permissions': (('can_create_users', 'Can Create User'), ('can_view_users', 'Can View User'), ('can_edit_users', 'Can Edit User'), ('can_remove_users', 'Can Remove User'), ('can_view_user_permissions', 'Can View User Permissions'), ('can_edit_user_permissions', 'Can Edit User Permissions'), ('can_read_all_user_data', 'Can Read All User Data'),
+('can_read_usernames', 'Can Read User Usernames'))},
```

Comment:
You shouldn't alter old migrations; you need to create new ones.  1) Undo the changes to this file. 2) You need to add 'can_read_usernames' to the[ User model permissions under the Meta class](https://github.com/CenterForOpenScience/lookit-api/blob/develop/accounts/models.py#L234); follow the patterns there.  3) Then run `python manage.py makemigrations`.  4. Commit new migration/model changes.